internal/cmds/cmds.go

```diff
@@ -117,22 +123,26 @@ func disable(ctx *log.Context, h types.HandlerEnvironment, report *types.RunComm
 }
 
 func install(ctx *log.Context, h types.HandlerEnvironment, report *types.RunCommandInstanceView, metadata types.RCMetadata, c types.Cmd) (string, string, error, int) {
+	extensionEvents := createExtensionEventManager(ctx, h)
 	exitCode, err := immediatecmds.Install()
 	if err != nil {
 		return "", "", err, exitCode
 	}
 
 	if err := os.MkdirAll(constants.DataDir, 0755); err != nil {
+		extensionEvents.LogErrorEvent("install", "Failed to create data dir")
```

Comment:
Please add the dataDir and also the error message why it failed (err.Error())

Reply:
added